### PR TITLE
feat: Campaign and Donation views

### PR DESCRIPTION
**Pull request by gabrielricardo7**

### CODE REVIEW

File: campaigns/urls.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    # EndCampaignView,
+    OngCampaignView,
+)
+
+urlpatterns = [
+    path("ongs/<str:ong_id>/campaigns/", OngCampaignView.as_view()),
+    path("campaigns/", CampaignView.as_view()),
+    path("campaigns/<str:campaign_id>/", CampaignIdView.as_view()),
+    path("campaigns/<str:campaign_id>/donate/", DonationView.as_view()),
+    # path("campaigns/<str:campaign_id>/end/", EndCampaignView.as_view()),
```
Comment (marcelosj3) [inline campaigns/urls.py:16]: Same as above

File: campaigns/urls.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from django.urls import path
+
+from .views import (
+    CampaignIdView,
+    CampaignView,
+    DonationView,
+    # EndCampaignView,
```
Comment (marcelosj3) [inline campaigns/urls.py:7]: Are you planning to implement these or can we delete those comments?

File: campaigns/views.py [hunk 3]
```diff
@@ -1,3 +1,149 @@
...
+
+            return Response(donation.data, status.HTTP_201_CREATED)
+
+        except Http404:
+            return Response(
+                {"details": "Campaign not found."}, status.HTTP_404_NOT_FOUND
+            )
+
+
+# class EndCampaignView(APIView):
```
Comment (marcelosj3) [inline campaigns/views.py:115]: Are we planning to use this?

File: sos_brazil/urls.py [hunk 4]
```diff
@@ -16,4 +16,9 @@
 from django.contrib import admin
 from django.urls import include, path
 
-urlpatterns = [path("admin/", admin.site.urls), path("api/", include("ongs.urls"))]
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("api/", include("campaigns.urls")),
+    path("api/", include("ongs.urls")),
+    # path("api/", include("users.urls")),
```
Comment (marcelosj3) [inline sos_brazil/urls.py:23]: Shouldn't it be available?

**Review by marcelosj3 — CHANGES_REQUESTED** [review]

File: sos_brazil/urls.py (hunk 4 above)
Comment (gabrielricardo7) [reply sos_brazil/urls.py:23]: Commented for not being implemented until the moment I tested in this branch…

File: campaigns/views.py (hunk 3 above)
Comment (gabrielricardo7) [reply campaigns/views.py:115]: Yeah, to make the "donate" endpoint…

File: campaigns/urls.py (hunk 2 above)
Comment (gabrielricardo7) [reply campaigns/urls.py:7]: Will be the donation endpoint import…

File: campaigns/urls.py (hunk 1 above)
Comment (gabrielricardo7) [reply campaigns/urls.py:16]: Will be the donation endpoint path…

**Review by gabrielricardo7 — COMMENTED** [review]
Comments and changes made…

**Review by marcelosj3 — APPROVED** [review]

**Review by gabrielsuch — APPROVED** [review]

**Pull request merged**